packages/venia-ui/lib/components/AccountMenu/accountMenuItems.js

```diff
@@ -32,7 +32,7 @@ const AccountMenuItems = props => {
                 onClick={handleSignOut}
                 type="button"
             >
-                <FormattedMessage id={`Sign Out`} />
+                <FormattedMessage id={`Sign Out`} defaultMessage={`Sign Out`} />
```

Comment:
Per discussion in dev time we should use id format like `accountMenu.signOutButtonText`